Podfile

```diff
@@ -15,6 +15,7 @@ target 'Lets Do This' do
     pod 'GoogleAnalytics', '3.13.0'
     pod 'NewRelicAgent', '5.3.4'
     pod 'NSString+RemoveEmoji', '0.1.0'
+    pod 'TapjoySDK'
```

Comment:
Adding TapjoySDK will download v11 -- our contact at Tapjoy has advised us we need to use v10 and we should be ok with http://cocoadocs.org/docsets/TapjoySDK/10.2.2/

However when I change our Podfile to specify 10.2.2 -- the Tapjoy `setDebugEnabled` and `connect` methods are not found:

![screen shot 2016-04-27 at 1 30 17 pm](https://cloud.githubusercontent.com/assets/1236811/14867214/57a0e188-0c7c-11e6-8803-33627dffebf7.png)